pipeline/outputs/loki.md

```diff
@@ -24,6 +24,7 @@ Be aware there is a separate Golang output plugin provided by [Grafana](https://
 | labels | Stream labels for API request. It can be multiple comma separated of strings specifying  `key=value` pairs. In addition to fixed parameters, it also allows to add custom record keys \(similar to `label_keys` property\). More details in the Labels section. | job=fluent-bit |
 | label\_keys | Optional list of record keys that will be placed as stream labels. This configuration property is for records key only. More details in the Labels section. |  |
 | label\_map\_path | Specify the label map file path. The file defines how to extract labels from each record. More details in the Labels section. | |
+| structured\_metadata | Optional comma separated list of `key=value` strings specifying structured metadata for the log line. Like the `labels` parameter, values may also reference record keys via record accessors. More details in the Structured Metadata section. | |
```

Comment:
```suggestion
| structured\_metadata | Optional comma-separated list of `key=value` strings specifying structured metadata for the log line. Like the `labels` parameter, values can reference record keys using record accessors. See [Structured metadata](#structured-metadata) for more information. | |
```

Reply:
Funny enough I originally wrote "comma-separated" as well, as that's clearly correct, but later removed the hyphen for consistency with the exist `labels` field which doesn't hyphenate it. :)